app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container.rb

```diff
@@ -24,6 +24,33 @@ class ManageIQ::Providers::Amazon::StorageManager::S3::CloudObjectStoreContainer
     end
   end
 
+  def self.params_for_create(_ems)
+    {
+      :fields => [
+        {
+          :component  => 'text-field',
+          :name       => 'name',
```

Comment:
for some reason, I expected this to be `bucket` so `:bucket` would come in on the `options` hash.